provider/ecs/ecs.go

```diff
@@ -554,6 +554,9 @@ func (p *Provider) getHost(i ecsInstance) string {
 }
 
 func (p *Provider) getPort(i ecsInstance) string {
+	if port := p.label(i, types.LabelBackendPort); port != "" {
```

Comment:
Could you please use [`types.LabelPort`](https://github.com/containous/traefik/blob/master/types/common_label.go#L10) instead of creating a new label.